b2sdk/api_config.py

```diff
@@ -26,7 +26,9 @@ def __init__(
         install_clock_skew_hook: bool = True,
         user_agent_append: str | None = None,
         _raw_api_class: type[AbstractRawApi] | None = None,
-        decode_content: bool = False
+        decode_content: bool = False,
+        expect_100_continue: bool = True,
+        expect_100_continue_timeout_seconds: float = 10.0,
```

Comment:
please rename to `expect_100_timeout` across the whole code base and make sure you added it to docstrings to methods which already have params in docstrings

Lets not do "expect 100 continue timeout" in one place and "expect 100 timeout"  in another. Lets have one and stick with it.